zuul.d/ansible-cloud-jobs.yaml

```diff
@@ -358,3 +358,30 @@
       - name: github.com/ansible-collections/community.vmware
     vars:
       ansible_collections_repo: github.com/ansible-collections/community.vmware
+
+
+### AWS
+- job:
+    name: ansible-test-cloud-integration-aws-py36
+    parent: ansible-core-ci-aws-session
```

Comment:
so, this is a little tricky. For here, you still need to parent to base-minimal or other 'ansible-test' jobs.

Then add a 2nd, entry
```
- job:
    name: ansible-test-cloud-integration-aws-py36
    parent: ansible-core-ci-aws-session
```
You basically with have 2 'ansible-test-cloud-integration-aws-py36' jobs, with 2 different parents.

Reply:
Another example:

  https://github.com/ansible/network-ee/blob/main/.zuul.d/jobs.yaml#L42
  https://github.com/ansible/network-ee/blob/main/.zuul.d/jobs.yaml#L76

single job, 2 parents.

Reply:
I actually don't need to inherit from these jobs.